include/experimental/kernel/gemm/impl/int4_dequantize_kslicing_xe.hpp

```diff
@@ -839,45 +372,54 @@ class gemm_universal_t<dispatch_policy_int4_dequantize_kslicing<group_swizzle_,
         bool implementable = true;
         if (gemm_t::msg_type_a != msg_type::unaligned_2d) {
             if (gemm_t::msg_type_a == msg_type::block_2d) {
-                implementable &= kernel::block_2d<gpu_arch::Xe,
-                        dtype_a>::check_tensor((uint64_t)(args.matA_base.base),
-                        gemm_t::is_col_major_a ? args.matrix_m : args.matrix_k,
-                        gemm_t::is_col_major_a ? args.matrix_k : args.matrix_m,
-                        args.matA_ld);
+                implementable
+                        &= kernel::block_2d<arch_tag, dtype_a>::check_tensor(
+                                (uint64_t)(args.matA_base.base),
+                                gemm_t::is_col_major_a ? args.matrix_m
+                                                       : args.matrix_k,
+                                gemm_t::is_col_major_a ? args.matrix_k
+                                                       : args.matrix_m,
+                                args.matA_ld);
             } else {
-                implementable &= kernel::general_1d<gpu_arch::Xe,
+                implementable &= kernel::general_1d<arch_tag,
```

Comment:
L128 change arch_tag to gpu_arch:Xe, why opposite here?

Reply:
gpu_arch::Xe should not seen in the code I have modified L128